config/crd/bases/postgres-operator.crunchydata.com_postgresclusters.yaml

```diff
@@ -5839,15 +5839,15 @@ spec:
                               with a pod if it doesn''t satisfy the spread constraint.
                               - DoNotSchedule (default) tells the scheduler not to
                               schedule it. - ScheduleAnyway tells the scheduler to
-                              schedule the pod in any location,   but giving higher
-                              precedence to topologies that would help reduce the   skew.
-                              A constraint is considered "Unsatisfiable" for an incoming
-                              pod if and only if every possible node assigment for
-                              that pod would violate "MaxSkew" on some topology. For
-                              example, in a 3-zone cluster, MaxSkew is set to 1, and
-                              pods with the same labelSelector spread as 3/1/1: |
-                              zone1 | zone2 | zone3 | | P P P |   P   |   P   | If
-                              WhenUnsatisfiable is set to DoNotSchedule, incoming
+                              schedule the pod in any location, but giving higher
+                              precedence to topologies that would help reduce the
+                              skew. A constraint is considered "Unsatisfiable" for
+                              an incoming pod if and only if every possible node assigment
+                              for that pod would violate "MaxSkew" on some topology.
+                              For example, in a 3-zone cluster, MaxSkew is set to
+                              1, and pods with the same labelSelector spread as 3/1/1:
+                              | zone1 | zone2 | zone3 | | P P P |   P   |   P   |
```

Comment:
📝 This is a particularly long and difficult [description to read](https://access.crunchydata.com/documentation/postgres-operator/v5/references/crd/#postgresclusterspecinstancesindextopologyspreadconstraintsindex) in plaintext. We might want to consider amending this field like we've done with some others.

Reply:
:+1: Do you have an example of a field we've amended?

Reply:
We removed a `TODO` buried within volume projections upstream: https://github.com/CrunchyData/postgres-operator/commit/28c2995189e63044d02432d56dd2f104c4181a2b

Reply:
Ah, I see now, yeah, I wouldn't describe that as a blocker, but I agree that it might be easier if we can link to something else that describes MaxSkew (and maybe WhenUnsatisfiable, that's another big chunk of topology-related text that's hard to read in this form).